base/threads_overloads.jl

```diff
@@ -20,6 +20,21 @@ to load-balancing. This approach thus may be more suitable for fine-grained,
 uniform workloads, but may perform worse than `FairSchedule` in concurrence
 with other multithreaded workloads.
 
+# Examples
+```julia-repl
+julia> n = 20
+
+julia> c = Channel{Int}(ch -> foreach(i -> put!(ch, i), 1:n), 1)
+
+julia> d = Channel{Int}(n) do ch
+           f = i -> put!(ch, i^2)
+           Threads.foreach(f, c)
+       end
+
+julia> collect(d)
+collect(d) = [1, 4, 9, 16, 25, 36, 49, 64, 81, 100, 121, 144, 169, 196, 225, 256, 289, 324, 361, 400]
```

Comment:
Because the number are sorted, this may give the impression the order of items in `d` might preserve the one in `c`, as in a neat parallel map, but I don't think it's the case. Only with `ntasks=1`.